PROTOCOL.md

```diff
@@ -0,0 +1,231 @@
+# Protocol Specification
+
+The ttrpc protocol uses simple frames for communicating messages between a
```

Comment:
In general I'd nix the word "simple", "easy", all around.

Maybe `The ttrpc protocol is a client/server protocol which uses request/response framing to allow multiple requests on a single stream.

Reply:
Updated